<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QWidget" name="MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widgetTitle" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>75</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="logoWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>145</width>
        <height>75</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgba(255,255,255,1);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetTop" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="9">
        <widget class="QToolButton" name="logoutBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>注销</string>
         </property>
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/img/index/nav_icon10.png</normaloff>:/img/index/nav_icon10.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="alarmBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>报警信息</string>
         </property>
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/img/index/nav_icon4.png</normaloff>:/img/index/nav_icon4.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QToolButton" name="configBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>系统配置</string>
         </property>
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/img/index/nav_icon9.png</normaloff>:/img/index/nav_icon9.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="currentDataBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>实时数据</string>
         </property>
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/img/index/nav_icon2.png</normaloff>:/img/index/nav_icon2.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QToolButton" name="statuBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>运行状态</string>
         </property>
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/img/index/nav_icon1.png</normaloff>:/img/index/nav_icon1.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="10">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="historyDataBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>历史数据</string>
         </property>
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/img/index/nav_icon3.png</normaloff>:/img/index/nav_icon3.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QToolButton" name="fingurePrintBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>指纹库</string>
         </property>
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/img/index/nav_icon5.png</normaloff>:/img/index/nav_icon5.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QToolButton" name="helpBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>系统帮助</string>
         </property>
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/img/index/nav_icon8.png</normaloff>:/img/index/nav_icon8.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QToolButton" name="userAdminBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>用户管理</string>
         </property>
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/img/index/nav_icon7.png</normaloff>:/img/index/nav_icon7.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QToolButton" name="selfCheckBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>自检</string>
         </property>
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/img/index/nav_icon6.png</normaloff>:/img/index/nav_icon6.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="11">
        <widget class="QWidget" name="infoWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>650</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,2,8,2,2,4,2,8">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="siteIconLabel">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../img.qrc">:/img/index/room_icon.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="siteNameLabel">
            <property name="styleSheet">
             <string notr="true">font-size:18px;
font-family:微软雅黑;
color:rgba(255,255,255,1);
line-height:24px;</string>
            </property>
            <property name="text">
             <string>大同500千伏变电站</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="userIconLabel">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../img.qrc">:/img/index/user_icon.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="userNameLabel">
            <property name="styleSheet">
             <string notr="true">font-size:18px;
font-family:微软雅黑;
color:rgba(255,255,255,1);
line-height:24px;</string>
            </property>
            <property name="text">
             <string>管理员</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="timeLabel">
            <property name="styleSheet">
             <string notr="true">font-size:18px;
font-family:微软雅黑;
font-weight:bold;
color:rgba(255,255,255,1);
line-height:24px;</string>
            </property>
            <property name="text">
             <string>2019年1月1日 00:00:00</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetMenu" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="articleWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>75</y>
     <width>1920</width>
     <height>1005</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../img.qrc"/>
 </resources>
 <connections/>
</ui>
